src/ui/templates/insurance/policy-and-exports/check-your-answers.njk

```diff
@@ -0,0 +1,52 @@
+{% extends 'index.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+
+{% block pageTitle %}
+  {{ CONTENT_STRINGS.PAGE_TITLE }}
+{% endblock %}
+
+{% block content %}
+
+  {{ govukBackLink({
+    text: CONTENT_STRINGS.LINKS.BACK,
+    href: BACK_LINK,
+    attributes: {
+      "data-cy": "back-link"
```

Comment:
should we maybe pass these data-cy values in page variables? tech debt? the common ones

Reply:
Great idea, added to backlog https://ukef-dtfs.atlassian.net/browse/EMS-833